src/textlint-filter-rule-allowlist.js

```diff
@@ -1,17 +1,30 @@
 import path from "path";
-import { rcFile } from "rc-config-loader";
+import yaml from "js-yaml";
 import { getConfigBaseDir } from "@textlint/get-config-base-dir";
 import { matchPatterns } from "@textlint/regexp-string-matcher";
 
+const loadAllowlistConfigFile = (baseDirectory, filePath) => {
+    const fs = require("fs");
+    const configFilePath = path.resolve(baseDirectory, filePath);
+    const extName = path.extname(configFilePath);
+    const configFile = fs.readFileSync(configFilePath);
+    let config = null;
+    if (extName == ".json") {
+        config = JSON.parse(configFile);
+    } else if (/\.(yml|yaml)/.test(extName)) {
```

Comment:
```suggestion
    } else if (/\.(yml|yaml)$/.test(extName)) {
```
Or, we can just write `extName === ".yml" || extName === ".yaml"`